### ISSUE TITLE
The discriminator loss

### ISSUE THREAD

**Issue by yzp12** [Yang7879/3D-RecGAN#1]
Hi! 
Thank you for sharing the code! I notice in the paper the output of the discriminator is a vector. I am wondering if the loss of the discriminator is the different between two sum of the output vector(one for real and one for fake)? 
Thanks!

**Comment by Yang7879** [comment]
@yzp12 Yes, the output of the discriminator is a vector instead of a single scalar. Mean value of that vector is the loss. For a real reconstruction pair, the mean value is optimized to be "1", otherwise, the mean value is pushed to "0".
